.azure-pipelines/london_taxi_ci_pipeline.yml

```diff
@@ -5,12 +5,12 @@ trigger:
      - development
   paths:
     include:
-      - .azure-pipelines/*
-      - mlops/common/*
-      - mlops/london_taxi/*
-      - src/london_src/*
-      - model/london_taxi/*
-      - src/shared/*
+      - .azure-pipelines/**
```

Comment:
my comment regarding two * was related to GitHub only. In AzDO it should be one star only